src/api-doc/mentoring-2.5-setup.md

```diff
@@ -366,7 +366,7 @@ USER_SERVICE_BASE_URL='/user/'
 BIG_BLUE_BUTTON_URL=https://dev.some.temp.org
 
 # Big blue button base URL
-BIB_BLUE_BUTTON_BASE_URL=/bigbluebutton/
+BIG_BLUE_BUTTON_BASE_URL=/bigbluebutton/
```

Comment:
@sumanvpacewisdom for older version of docs, we should't make any env changes, the docker images dependent on those env keys only